Makefile.in

```diff
@@ -111,10 +111,10 @@ dist:	$(FILES) $(DISTSUBDIRS)
 	tar czf $(package).tar.gz $(DISTDIR)
 
 install:	$(LIBS) $(HEADERS) $(SUBDIRS)
-	test -d $(libdir) || mkdir -p $(libdir)
-	test -d $(includedir) || mkdir -p $(includedir)
-	$(install) $(LIBS) $(libdir)/
-	$(install) $(HEADERS) $(includedir)/
+	test -d $(DESTDIR)$(libdir) || mkdir -p $(DESTDIR)$(libdir)
```

Comment:
I do not understand why `DESTDIR` would be used here and at the other places specified. The unfolding of `libdir` and `includedir` follows the common GNU autoconf pattern to:

```
exec_prefix = ${prefix}
libdir = ${exec_prefix}/lib
includedir = ${prefix}/include/tinydtls
```

Given that `prefix` can be set to contain whatever you might want to set `DESTDIR` to, I do not see why you need another prefix?

Reply:
More information on `DESTDIR` can be found here: 
- https://www.gnu.org/prep/standards/html_node/DESTDIR.html
- https://www.gnu.org/software/automake/manual/html_node/DESTDIR.html

Basically, `DESTDIR` is used to set the path where the different component will be installed on the _local_ machine (e.g. `/home/fabrice/buildroot/output/target/`) whereas prefix and exec_prefix must be set to the path on the _target_ machine (e.g. `/usr`). As explained in the second link above: 

> DESTDIR support is strongly recommended in GNU packages, though it is not an absolute requirement.

Reply:
Sorry, I have missed this.